### PR TITLE
RD-3912 Update cfy plugins documentation

### PR DESCRIPTION
**Pull request by mateumann**

### CODE REVIEW

File: content/cli/orch_cli/plugins.md [hunk 1]
```diff
@@ -280,6 +280,72 @@ Plugin `e90b1a09-6b56-4a92-b9cd-5fc4ef32ab74` is now owned by user `admin`.
 ...
 {{< /highlight >}}
 
+
+### blueprint-labels
+
+#### blueprint-labels list
+`cfy plug blueprint-labels list [OPTIONS] PLUGIN_ID`
```
Comment (geokala) [inline content/cli/orch_cli/plugins.md:287]: We should probably use the full form (cfy plugins blueprint-labels...) in docs.

File: content/cli/orch_cli/plugins.md [hunk 2]
```diff
@@ -280,6 +280,72 @@ Plugin `e90b1a09-6b56-4a92-b9cd-5fc4ef32ab74` is now owned by user `admin`.
...
+
+#### blueprint-labels add
+`cfy plug blueprint-labels add [OPTIONS] LABELS_LIST PLUGIN_ID`
+
+Add blueprint labels for specific plugin.
+
+`PLUGIN_ID` is the id of the plugin.
+
+`LABELS_LIST` is a list in `<label1>:<value1>,<label2>:<value2>` format of labels.  Any comma and
+colon in `<value>` must be escaped with `\`.
```
Comment (geokala) [inline content/cli/orch_cli/plugins.md:301]: And backslash values?

**Comment by geokala** [comment]
One other query: Do we note anywhere what blueprint and deployment labels for plugins actually are?

File: content/cli/orch_cli/plugins.md (hunk 1 above)
Comment (mateumann) [reply content/cli/orch_cli/plugins.md:287]: Oh, right

File: content/cli/orch_cli/plugins.md (hunk 2 above)
Comment (mateumann) [reply content/cli/orch_cli/plugins.md:301]: They work fine, e.g. `cfy plug blueprint-labels add foo:/ ce5562c6-6e35-49ae-a74e-4f3e0b98e63c` will produce

```
Listing blueprint_labels of plugin ce5562c6-6e35-49ae-a74e-4f3e0b98e63c...

Plugin labels
+---------------+----------------------------------+
|      key      |              values              |
+---------------+----------------------------------+
|      foo      |               "/"                |
+---------------+----------------------------------+
```

Comment (geokala) [reply content/cli/orch_cli/plugins.md:301]: That's a forwardslash. What happens if I want foo:\

**Comment by mateumann** [comment]
blueprint and deployment labels are described in other parts of the docs (mostly here: https://docs.cloudify.co/latest/developer/blueprints/spec-labels/, https://docs.cloudify.co/latest/developer/blueprints/spec-blueprint-labels/)

**Comment by geokala** [comment]
Great- just the slash discussion then. <smiles>

File: content/cli/orch_cli/plugins.md (hunk 2 above)
Comment (mateumann) [reply content/cli/orch_cli/plugins.md:301]: `cfy plug blueprint-labels add bar:\\ ce5562c6-6e35-49ae-a74e-4f3e0b98e63c`

```
Listing blueprint_labels of plugin ce5562c6-6e35-49ae-a74e-4f3e0b98e63c...

Plugin labels
+---------------+----------------------------------+
|      key      |              values              |
+---------------+----------------------------------+
|      foo      |               "/"                |
|      bar      |               "\"                |
+---------------+----------------------------------+
```

**Review by geokala — APPROVED** [review]

**Pull request merged**